ui/src/app/shared/components/vars-input/vars-input.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import * as ReactForm from 'react-form';
+
+/*
+This is a lift-and-shift of env-input.
```

Comment:
Please reuse ArrayInput. You might use generic to support items type and support custom input renderer. e.g.:

```
export const ArrayInput<T> = (props: {items: []T, renderInput: (item: T): ReactNode ) {

```

Reply:
I was scared by genercising it , I thought i would make them too  complex. Lets discus IRL.